special-widget.scss

```diff
@@ -0,0 +1,17 @@
+.special-widget {
+	background: grey;
+	row: 1;
+ 
+	& > .content--red {
```

Comment:
The BEM modifier should probably be named according to statefulness rather than appearance. e.g. `content--alert` might be a better choice of class name if the intention of the red background is to display the widget in an alert state.

Reply:
It might be worth considering the benefits of placing this modifier on the parent element so that all child elements can benefit from it, e.g.

```
.special-widget {
  ...
  $.special-widget--alert {
    > .content {
      background: red;
    }

    // Style other child elements when the widget is in an alert state
  }
}
```